src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Area {
+    protected static final DecimalFormat df = new DecimalFormat("#.00");
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public abstract double getArea();
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract void draw();
```

Comment:
Similar to 'getArea', the 'draw' method could also be part of a separate interface, as 'draw' and 'getArea' are conceptually different and should not be in a single interface.